PodBinaryCacheExample/DevPods/ConfigSDK/ConfigSDK.podspec

```diff
@@ -16,5 +16,5 @@ Pod::Spec.new do |s|
   s.source_files = 'Classes/**/*'
 
   s.dependency 'SQLite.swift'
-  s.dependency 'ProtocolBuffers-Swift'
+  # s.dependency 'ProtocolBuffers-Swift'
```

Comment:
Can we discard the changes in `PodBinaryCacheExample` (if not required) to make the diff more relevant and easy to review?